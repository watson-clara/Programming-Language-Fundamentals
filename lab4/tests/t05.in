(difference '(1) '(1))
(difference '(1 2 3 4) '(3 4 5 6))
(difference '(a b c d) '(b d e))
(difference '(b d e) '(a b c d))
(difference '(patience you must have) '(my young padawan))
