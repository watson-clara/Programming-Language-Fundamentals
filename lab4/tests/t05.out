'()
'(1 2)
'(a c)
'(e)
'(patience you must have)
